### PR TITLE
Step 5 completed for dm feature

### PR DESCRIPTION
**Pull request by sebrianne**
The instructions just requested that I change the .js file, but I needed to add a 'recipient' field to getAllMessages() in datastore.java.

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -107,8 +107,9 @@ public int getTotalMessageCount(){
         String user = (String) entity.getProperty("user");
         String text = (String) entity.getProperty("text");
         long timestamp = (long) entity.getProperty("timestamp");
+        String recipient = (String) entity.getProperty("recipient");
 
-        Message message = new Message(id, user, text, timestamp);
+        Message message = new Message(id, user, text, timestamp, recipient); //this line was causing an error
```
Comment (PandiyanA) [inline src/main/java/com/google/codeu/data/Datastore.java]: //this line was causing an error
What is this comment for?

**Review by PandiyanA — COMMENTED** [review]
Thanks for making progress and also keeping the PR small!.

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (sebrianne) [reply src/main/java/com/google/codeu/data/Datastore.java]: I was trying to explain why I changed it. I know Joel added these lines but I needed to change them to work for my step 5.

**Review by PandiyanA — APPROVED** [review]
Thanks for addressing comments.

**Pull request merged**